cpu/samd21/cpu.c

```diff
@@ -32,7 +32,7 @@
 #endif
 
 #ifndef GEN3_ULP32K
-#define GEN3_ULP32K         1
```

Comment:
related ?

Reply:
Yes, that was the original change introduced by  #13306 hat  51fa5afef7a0d3b220563468dc3af68c50e97bb0 undoes.